### DISCUSSION TITLE
QUERY_THREAD_TIMEOUT isn't working for automations

### DISCUSSION THREAD

**Discussion by enpro-github** [Budibase/budibase#13017] [Help] [answered]
**Hosting**

- Self
  - Method: docker-compose
  - Budibase Version: 2.13.5

Like the title says, my automations are still timing out after like 12 seconds, but all my other SQL queries not used in automations have timeouts reflected by the QUERY_THREAD_TIMEOUT env variable. Is this supposed to be the case?

Thank you

**Comment by ConorWebb96** [comment, marked as answer]
Hey @enpro-github,

You will be able to do this whenever this pr is merged. An engineer added an new environment variable that will solve this for self-hosted users. https://github.com/Budibase/budibase/pull/13032

**Reply by enpro-github** [reply]
Late on the reply but thank you
